apps/vr-tests/src/stories/Breadcrumb.stories.tsx

```diff
@@ -9,8 +9,14 @@ storiesOf('Breadcrumb', module)
   .addDecorator(FabricDecoratorTall)
   .addDecorator(story => (
     <Screener
-      steps={new Screener.Steps()
+      steps={new Steps()
```

Comment:
Leave as it was before and remove the `Steps` import. If it was working as is before that means the import was unnecessary.